test/demo.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: The RamenDR authors
```

Comment:
This file should be now at `test/envs/` and we need to document it in the
[README](https://github.com/RamenDR/ramen/blob/main/test/README.md).